atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -139,4 +137,12 @@ public void compactInternally() {
             }
         }
     }
+
+    private String prefixedTableName() {
```

Comment:
nit: these are duped.  I would rather have this logic (however simple) consistent across all places it is used.  I would argue that it should be made available as a util method use across these classes.